docs/config.md

```diff
@@ -502,10 +502,14 @@ may be undesirable, e.g.:
 - There are multiple remotes with conflicting views of that branch,
   resulting in an unhelpful conflicted state.
 
-You can disable this behavior by setting `git.auto-local-branch` like
-so,
+You can disable this behavior by setting `git.auto-local-branch` like so,
 
-    git.auto-local-branch = false
+```toml
+git.auto-local-branch = false
+```
+
+This setting is applied only to new remote branches. Existing remote branches
+can be tracked individually.
```

Comment:
nit: move to last commit in the PR since there doesn't seem to be a way of tracking branches individually until then?